simple_email_confirmation/models.py

```diff
@@ -24,16 +24,8 @@ class SimpleEmailConfirmationUserMixin(object):
     Provides python-level functionality only.
     """
 
-    # if your User object stores the User's primary email address
-    # in a place other than User.email, you can override the
-    # primary_email_field_name and/or primary_email get/set methods.
-    # All access to a User's primary_email in this app passes through
-    # these two get/set methods.
-
-    primary_email_field_name = 'email'
```

Comment:
To avoid a regression for people using this field, we could do:

```python
def primary_email_field_name(self):
    "DEPRECATED. Use EMAIL_FIELD instead of primary_email_field_name to set the primary email field name"
     if hasattr(self, 'primary_email_field_name'):
        # maybe a warning here ?
        return self.primary_email_field_name
     return super(SimpleEmailConfirmationUserMixin, self).primary_email_field_name()
```

Just an idea, thanks for the PR, it's again good stuff